control-plane/connect-inject/endpoints_controller.go

```diff
@@ -380,7 +458,8 @@ func (r *EndpointsController) upsertHealthCheck(pod corev1.Pod, client *api.Clie
 func getServiceName(pod corev1.Pod, serviceEndpoints corev1.Endpoints) string {
 	serviceName := serviceEndpoints.Name
 	// If the annotation has a comma, it is a multi port Pod. In that case we always use the name of the endpoint.
-	if serviceNameFromAnnotation, ok := pod.Annotations[annotationService]; ok && serviceNameFromAnnotation != "" && !strings.Contains(serviceNameFromAnnotation, ",") {
+	if serviceNameFromAnnotation, ok := pod.Annotations[annotationService]; ok && serviceNameFromAnnotation != "" &&
+		!strings.Contains(serviceNameFromAnnotation, ",") {
```

Comment:
I think it's more clear which part is an if statement if you format it this way:

```suggestion
	if serviceNameFromAnnotation, ok := pod.Annotations[annotationService]; 
		ok && serviceNameFromAnnotation != "" && !strings.Contains(serviceNameFromAnnotation, ",") {
```